### ISSUE TITLE
Short freeze/period of lag during transitions or opening map/inventory

### ISSUE THREAD

**Issue by rubberfistfight** [elishacloud/Silent-Hill-2-Enhancements#102]
So the game runs fantastically, seems that everything is working as intended...

However, when being teleported ingame (from one side of gate to another, in the well etc)  or when accessing the map or the inventory there is a noticeable "freeze" before the transition triggers, almost like it's loading something. All animation including the noise filter stop during this period, it's about 2.5 seconds. The game is installed on a Sata 3 SSD so it cannot possibly be a loading lag.

The transitions THEMSELVES are faster than vanilla, so I know that's working. I've tried disabling the reShade stuff, it makes no difference either way. I updated reShade as well to no effect. 

I've also verified that it's independent of resolution and framerate settings, as well as not being affected by the CPU affinity and fast transition settings. It's not a problem of my system being unable to render the game - it's a Ryzen 5 2600 cpu coupled with a Vega 64 video card - it's more than up to the task.

So what could be causing this?

**Comment by Polymega** [comment]
This is a quirk with AMD GPUs when playing I'm fullscreen mode. From the [troubleshooting](http://enhanced.townofsilenthill.com/SH2/#troubleshooting) part of the guide:

Issue | OS | GPU | Recommendation
-- | -- | -- | --
The FastTransitions feature is noticeably delayed while playing in fullscreen mode. | Any | AMD | Set `EnableWndMode = 1`, `WndModeBorder = 0`, `ResX = 0`, and `ResY = 0` in the `d3d8.ini` file.

In short: Windowed mode must always be on for AMD cards (`EnableWndMode = 1`). Following the steps above, the game will play in windowed mode to alleviate your issue but still take up the entire display of your screen.

In a future update, we will have the game play in windowed mode by default (from the d3d8.ini file) so AMD GPU users won't have this issue "out of the box" when using the mod.

**Comment by rubberfistfight** [comment]
Ooh, you're right, I did not see that. Thank you for pointing it out so quickly.

I will test this out next time I play the game.

**Comment by rubberfistfight** [comment]
Just tested this and it works perfectly. Thank you for the pointer and sorry for wasting your time.

Now the only thing I'm hoping for is a way to get vibration to work with Xinput Plus :D

**Comment by Polymega** [comment]
You're not wasting our time; we're here to help the best we're able!

And vibration will be restored for the next update to this project: https://github.com/elishacloud/Silent-Hill-2-Enhancements/pull/99

**Comment by elishacloud** [comment]
Closing.  Issue is resolved.
